manifests/templates/disks-images-provider.yaml.in

```diff
@@ -0,0 +1,92 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: host-path-disk-datavolume1
+  labels:
+    kubevirt.io: ""
+    os: "datavolume1"
+spec:
+  capacity:
+    storage: 1Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: /tmp/hostImages/datavolume1
+  persistentVolumeReclaimPolicy: Recycle
```

Comment:
Do you really want recycle? Also, AFAIK recycle will be deprecated (though that's been a long road).

Reply:
the PVs are static in this setup, so we'll need recycling to use them with multiple PVCs throughout the test suite.

Reply:
The local volume PVs from kubevirtci are also set to recycle.